recipes-devtools/gie/tensorrt-plugins-prebuilt_8.0.1-1.bb

```diff
@@ -57,28 +42,27 @@ do_configure() {
 }
 
 do_compile() {
-    find ${S}/usr/src/tensorrt -name '*.py' | xargs sed -i -r -e 's,^(\s*)print "(.*)$,\1print("\2),'
+    :
 }
 
 do_install() {
     install -d ${D}${includedir}
     install -m 0644 ${S}/usr/include/aarch64-linux-gnu/*.h ${D}${includedir}
     install -d ${D}${libdir}
-    tar -C ${S}/usr/lib/aarch64-linux-gnu -cf- . | tar -C ${D}${libdir}/ --no-same-owner -xf-
+    cp --preserve=mode,timestamps,links --no-dereference ${S}/usr/lib/aarch64-linux-gnu/*.so* ${D}${libdir}
     install -d ${D}${prefix}/src
     cp --preserve=mode,timestamps --recursive ${S}/usr/src/tensorrt ${D}${prefix}/src/
 }
-PACKAGES += "${PN}-samples"
+
 FILES:${PN} += "${prefix}/src/tensorrt/bin"
-FILES:${PN}-samples = "${prefix}/src"
 
-RDEPENDS:${PN} += "libstdc++ cudnn libcublas cuda-cudart cuda-nvrtc cuda-nvtx tegra-libraries libglvnd"
-RDEPENDS:${PN}-samples += "tegra-libraries bash python3 libglvnd cudnn cuda-cudart libcublas"
-RPROVIDES:${PN}-samples = "${PN}-examples"
+RDEPENDS:${PN} += "tensorrt-core"
```

Comment:
I think `tegra-libraries` is also needed here, since `libnvinfer_plugin.so` does a `dlopen()` on the `libcuda.so` provided there.  `tensorrt-core` shouldn't be needed, since it will get pulled in due to linking against `libnvinfer.so`.

Reply:
Agree